src/library_webgpu.js

```diff
@@ -502,9 +509,11 @@ var LibraryWebGPU = {
   // wgpuDevice
 
   wgpuDeviceGetDefaultQueue: function(deviceId) {
-    assert(WebGPU.mgrQueue.objects.length === 1, 'there is only one queue');
-    var device = WebGPU["mgrDevice"].get(deviceId);
-    return WebGPU.mgrQueue.create(device["defaultQueue"]);
+    if(WebGPU.mgrQueue.objects.length === 1) {
```

Comment:
This is a nice solution.

`=== 1` should work for the first device, but I think we can easily make this work for multiple devices now, with something like:
```suggestion
    if (WebGPU.defaultQueues[deviceId] === undefined) {
```